herbert_core/utilities/maths/noisify.m

```diff
@@ -50,8 +56,13 @@
         if ~ymax_calculated
             ymax = max(abs(y(:)))
         end
-        % make noise dy and add to y for output; make error bar for noise    
-        dy=(fac*ymax)*randn(size(y));   % st. dev. of randn is sigma=1
+        % make noise dy and add to y for output; make error bar for noise.  
+        % randfunc generates the random numbers producing the noise;
+        % its standard and default implementation is randn - gaussian of std.dev=1
+        % but other choices may be appropriate including a non-random
+        % sequence to check functionality. Not using randn may cause
+        % unexpected behaviour from the user point of view
```

Comment:
```suggestion
        % unexpected behaviour from the user's point of view
```

Reply:
Your correction is acceptable, but using "user",  as an adjective qualifying point, seems perfectly good to me. Compare "user experience" vs "user's experience".